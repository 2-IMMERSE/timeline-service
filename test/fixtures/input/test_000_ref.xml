<tl:document xmlns:tl="http://jackjansen.nl/timelines" xmlns:tim="http://jackjansen.nl/2immerse" xmlns:tic="http://jackjansen.nl/2immerse/component" title="Hello World document, similar to previously hardcoded timelineServer example">
    <tl:par tl:end="master">
        <tl:ref tim:class="video"
                xml:id="masterVideo"
                tic:syncMode="master"
                tic:mediaUrl="../media/bunny.mp4"
                />
    </tl:par>
</tl:document>
