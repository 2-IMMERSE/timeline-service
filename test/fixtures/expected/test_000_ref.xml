<?xml version='1.0' encoding='utf8'?>
<tl:document xmlns:tic="http://jackjansen.nl/2immerse/component" xmlns:tim="http://jackjansen.nl/2immerse" xmlns:tl="http://jackjansen.nl/timelines" xmlns:tls="http://jackjansen.nl/timelines/internal" title="Hello World document, similar to previously hardcoded timelineServer example" tls:slavedProgress="42.345">
    <tl:par tls:slavedProgress="42.345" tl:end="master">
        <tl:ref tic:mediaUrl="../media/bunny.mp4" tic:syncMode="master" tim:class="video" xml:id="masterVideo" />
    </tl:par>
</tl:document>